cypress/integration/requests/request-types-filter.spec.js

```diff
@@ -0,0 +1,99 @@
+import uuid from 'uuid';
+import TestTypes from '../../support/dictionary/testTypes';
+import Requests from '../../support/fragments/requests/requests';
+import TopMenu from '../../support/fragments/topMenu';
+import { Checkbox, Pane } from '../../../interactors';
+
+describe('ui-requests: Make sure that request type filters are working properly', () => {
+  const requests = [];
+  const instances = [];
+  const userIds = [];
+  const resetFiltersMessage = 'Choose a filter or enter a search query to show results.';
+  const doesNotExistRequest = `notExist-${uuid()}`;
+  const getNoResultMessage = term => `No results found for "${term}". Please check your spelling and filters.`;
+
+  before(() => {
+    cy.login(Cypress.env('diku_login'), Cypress.env('diku_password'));
+    cy.getToken(Cypress.env('diku_login'), Cypress.env('diku_password'));
+  });
+
+  beforeEach(() => {
+    Object.values(Requests.requestTypes).forEach(requestType => {
+      const itemStatus = requestType === 'Page' ? 'Available' : 'Checked out';
+      Requests
+        .createRequestApi(itemStatus, requestType)
+        .then(({ instanceRecordData, createdRequest, createdUser }) => {
+          requests.push(createdRequest);
+          instances.push(instanceRecordData);
+          userIds.push(createdUser.id);
+        });
+    });
+  });
+
+  after(() => {
+    instances.forEach(instance => {
+      cy.deleteItem(instance.itemId);
+      cy.deleteHoldingRecord(instance.holdingId);
+      cy.deleteInstanceApi(instance.instanceId);
+    });
+    requests.forEach(request => {
+      Requests.deleteRequestApi(request.id);
+    });
+    userIds.forEach(id => {
+      cy.deleteUser(id);
+    });
+  });
+
+  it('C540 Make sure that request type filters are working properly', { tags: [TestTypes.smoke] }, () => {
```

Comment:
test in testrail is not special and should be redesigned to special steps and expectations. I suggest to use your implementation/ May be test should be splitted to separated parts